artist/field/receiver_field.py

```diff
@@ -61,6 +63,7 @@ def from_hdf5(cls, config_file: h5py.File) -> Self:
             Receiver.from_hdf5(
                 config_file=config_file[config_dictionary.receiver_key][receiver_name],
                 receiver_name=receiver_name,
+                device=device,
```

Comment:
The `device` needs to be converted to a `torch.device` in the beginning of this function (I could not make a commit suggestion for this line). Alternatively, if this is the only occurrence, convert directly in the function:
```suggestion
                device=torch.device(device),
```

Reply:
done